sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/models/PhoneNumberErrorResponseException.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.phonenumbers.models;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.http.HttpResponse;
+
+/**
+ * Exception thrown on phone number specific errors
+ */
+public class PhoneNumberErrorResponseException extends HttpResponseException {
+    /**
+     * Initializes a new instance of the PhoneNumberErrorResponseException class.
+     *
+     * @param message the exception message or the response content if a message is not available.
+     * @param response the HTTP response.
+     */
+    public PhoneNumberErrorResponseException(String message, HttpResponse response) {
```

Comment:
Why do we need this constructor?

Reply:
You're right, we are not using it anywhere